compose/ui/ui-test-junit4/src/desktopTest/kotlin/androidx/compose/ui/test/TestBasicsTest.kt

```diff
@@ -181,4 +187,68 @@ class TestBasicsTest {
         rule.unregisterIdlingResource(idlingResource)
         test(expectedValue = "first")
     }
+
+    @Test(timeout = 500)
+    fun infiniteLoopInLaunchedEffectDoesNotHang() = runComposeUiTest {
```

Comment:
Can these new tests be in commonTest? 

Is it in desktopTest because you use `@Test(timeout=...)` ?

As I understand if one of the tests is stuck, it will eventually throw a timeout anyway. So perjaps we can rely on the default Timeout?

Reply:
I'll see if I can move these tests to common/skiko in a separate PR.